Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -463,20 +454,35 @@ void ABenchmarkGymGameModeBase::TickActorCountCheck(float DeltaSeconds)
 {
 	const UNFRConstants* Constants = UNFRConstants::Get(GetWorld());
 	check(Constants);
-	FMetricTimer TempActorCheckDelay = Constants->ActorCheckDelay;
 	// This test respects the initial delay timer in both native and GDK
-	if (TempActorCheckDelay.HasTimerGoneOff() &&
+	if (Constants->ActorCheckDelay.HasTimerGoneOff() &&
+		TickActorCountTimer.HasTimerGoneOff() &&
```

Comment:
TickActorCountTimer was added to allow us to set the frequency at which we wish to report actor count data. 
Preferable to re-using the Constants->ActorCheckDelay as these are "constant" timers (timers that should not really be modified). We could technically re-use it as the old code attempted to do...